### DISCUSSION TITLE
Happy New Year 2024!

### DISCUSSION THREAD

**Discussion by cressidagp** [arcemu/arcemu#453] [General]
Greetings,

I just wanted to wish a good year for all ArcEmu community members, stalkers and developers. Especially to ArcEmu Legendary Dev @dfighter1985 

Thanks for all the hard work You did this year. You're the Best!

**Comment by dfighter1985** [comment]
I just got back home. It's dangerous outside, people partying on the street, with fireworks, crackers, and whatnot, and if you are not careful they make you drink alcohol, and the tram tries to run over you too!

![IMG_20240101_001825](https://github.com/arcemu/arcemu/assets/589414/1d0ca6bc-9b03-4af2-b12e-b9244c924cc5)
![IMG_20240101_002934](https://github.com/arcemu/arcemu/assets/589414/13ea43a1-7986-4de2-a506-988982ef59be)

https://www.youtube.com/watch?v=Rx1rtxeHZ28

(Here's the location in case you liked the view: https://www.google.com/maps/@47.4952923,19.0485338,3a,75y,283.93h,87.63t/data=!3m6!1e1!3m4!1sXnh9oXsrxAlf0XwrDN5EDA!2e0!7i13312!8i6656?entry=ttu )

So don't go there! Keep on working on Arcemu, it's much safer! :D

I am just kidding, y'all should have fun!
Happy new year everyone!
